connectors/es/sink.py

```diff
@@ -170,6 +179,48 @@ async def _bulk_api_call():
 
         return res
 
+    async def _log_bulk_operations(self, res):
+        for item in res.get("items", []):
+            if "index" in item:
+                action_item = "index"
+            elif "delete" in item:
+                action_item = "delete"
+            elif "create" in item:
+                action_item = "create"
+            elif "update" in item:
+                action_item = "update"
+            else:
+                # Should only happen, if the _bulk API changes
+                # Unlikely, but as this functionality could be used for audits we want to detect changes fast
+                self._logger.error(
+                    f"Unknown action item returned from _bulk API for item {item}"
+                )
+                return
```

Comment:
should this `continue` instead?

Reply:
Oh yes of course, good catch! Adjusted with [153460d](https://github.com/elastic/connectors/pull/2117/commits/153460d660bd13673edc6aef960652bb19a725af)